README.md

```diff
@@ -6,9 +6,9 @@ Jamulus - Internet Jam Session Software
 =======================================
 <a href="https://jamulus.io/"><img align="left" src="src/res/homepage/mediawikisidebarlogo.png"/></a>
 
-The Jamulus software enables musicians to perform real-time jam sessions over the internet.
-There is one server running the Jamulus server software which collects the audio data from
-each Jamulus client, mixes the audio data and sends the mix back to each client.
+Jamulus enables musicians to perform in real-time together over the internet.
+A Jamulus server collects the audio data from
```

Comment:
Minor suggestion to increase clarity: "the audio data" -> "the incoming audio data".